sullivan_a10.js

```diff
@@ -0,0 +1,22 @@
+/*  Anna Sullivan
+    sullivan_a10.js
+    INFO2124WW
+    Thoendel
+    2/22/2020
+*/
+
+
+document.getElementById('action').addEventListener('click', function() {
+    //WRITE YOUR SOLUTION BETWEEN THIS COMMENT
+    
+let img = document.getElementById("photo"); //return the element photo by its id
+
+img.getAttribute("src"); //get attribute src from the photo element

```

Comment:
this really isn't doing anything since you aren't storing it or checking it in anyway.